src/components/ui/CustomSelect/SearchSelect.jsx

```diff
@@ -165,7 +165,11 @@ export default class CustomSelect extends React.Component {
           <ul className="custom-select-options">
             {this.state.filteredOptions.map((item, index) => {
               const cName = (index === this.state.selectedIndex) ? '-selected' : '';
-              return <li className={cName} key={index} onMouseEnter={() => { this.setSelectedIndex(index); }} onMouseDown={() => this.selectItem(item)}>{item.label}</li>;
+              return (
+                <li className={cName} key={index} onMouseEnter={() => { this.setSelectedIndex(index); }} onMouseDown={() => this.selectItem(item)}>
```

Comment:
- Please split its properties into different lines, for human readability....
- Why are we using onMouseDown?? Shouldn't we use onClick?